### PR TITLE
OCPBUGS32411: Updated prerequisites for oc-mirror plugin

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.14+

Issue: [OCPBUG32411](https://issues.redhat.com/browse/OCPBUGS-32411)

Link to docs preview: [Preview](https://75071--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.html)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oc-mirror-installing-plugin.adoc [hunk 1]
```diff
@@ -12,6 +12,8 @@ To use the oc-mirror OpenShift CLI plugin to mirror registry images, you must in
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
+* Make sure that the `umask` is set to `0022` on the operating system that uses oc-mirror.
+* Make sure you are using the correct `RHEL8` binary if your operating system is `RHEL8`, and `RHEL9` for `RHEL9` if your operating system is `RHEL9`. As `RHEL9` is FIPS compliant and has different linking for static and dynamic libraries.
```
Comment (kasturinarra) [inline modules/oc-mirror-installing-plugin.adoc:16]: @subhtk should we change like `Make sure you are using the correct `RHEL8` binary if your operating system is `RHEL8` and `RHEL9` binary if your operating system is `RHEL9` ? Not sure if `and `RHEL9` for `RHEL9`` is okay.

**Review by xenolinux — COMMENTED** [review]
I left some comments for consideration.

I suggest using the same structure/tense for all of the prerequisites.
For example:
* You have set the `umask`....
* You have installed....

Can you please confirm the applicable OCP versions? Per jira ticket it should be 4.14+.
Gentle reminder to add a preview link and update QE approval check box.

File: modules/oc-mirror-installing-plugin.adoc [hunk 2]
```diff
@@ -12,6 +12,8 @@ To use the oc-mirror OpenShift CLI plugin to mirror registry images, you must in
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
+* Make sure that the `umask` is set to `0022` on the operating system that uses oc-mirror.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/oc-mirror-installing-plugin.adoc:15]: Ack.

**Pull request merged**